app_kitchen/templates/item_new.html

```diff
@@ -6,10 +7,55 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
 </head>
 
 <body>
-  new items page
+  <container>
+    <div class="text-danger">
+      {%if messages%}
+      <ul class="list-unstyled d-inline-block">
+        {%for message in messages%}
+        <li class="text-left">
+          {{ message }}
+        </li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+    </div>
+  </container>
+  <div class="header">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm">
+          <div class="card shadow">
+            <div class="card-header">
+              <h1>Add a New Item!</h1>
+              <form action="/items/" method="POST">
+                {%csrf_token%}
+                <div class="card-body"></div>
+                <div class="form-group">
+                  <label>Item Name :<input type="text" name="name" id=""></label>
+                  <div class="form-group">
+                    <label>Item Image Link: <input type="text" name="image_link" id="" class="form-control">
+                    </label>
+                  </div>
+                  <div class="form-group">
+                    <label>Item Quantity<input type="text" name="quantity" id="" class="form-control"></label>
+                  </div>
+                  <button type="submit" class="btn-danger">Submit</button>
+              </form>
+            </div>
+          </div>
+        </div>
+        <div class="col-sm">
```

Comment:
Do these divs provide a specific purpose or can they be omitted?

Reply:
done